components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -14,8 +14,8 @@ import java.lang.ref.WeakReference
  * WebView-based EngineSession implementation.
  */
 class SystemEngineSession : EngineSession() {
-    internal var urlToLoad: String? = null
     internal var view: WeakReference<SystemEngineView>? = null
+    internal var scheduledLoad = ScheduledLoad()
```

Comment:
I put it as an empty constructed data class to avoid the extra `?` in it's usage which I think would decrease our code coverage:
```kotlin
internalSession.scheduledLoad?.data?.let {
  //...
}
```

I've changed the assignment to make it more explicit:
```kotlin
internal var scheduledLoad = ScheduledLoad(null)
```